03_homework/homework.sh

```diff
@@ -7,21 +7,58 @@
 # On your terminal, input all the commands you have used to create the following:
 
 # 1. How would you create 5 directories? Feel free to use any name for your directories.
+mkdir dir1 dir2 dir3 dir4 dir5
 
 # 2. How would you verify the creation of all 5 directories?
+ls -F
 
 # 3. In each directory, how would you create 5 .txt files and write "I love data" into each within the directories?
 
+touch dir1/data1.txt
+echo "I love data" >> dir1/data1.txt
+touch dir2/data2.txt
+echo "I love data" >> dir2/data2.txt
+touch dir3/data3.txt
+echo "I love data" >> dir3/data3.txt
+touch dir4/data4.txt
+echo "I love data" >> dir4/data4.txt
+touch dir5/data5.txt
+echo "I love data" >> dir5/data5.txt
```

Comment:
Remember the difference between `>` and `>>`, note that you are trying to write `I love data` into these files